app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,70 @@
+class NotificationsController < ApplicationController
+  skip_before_action :verify_authenticity_token
```

Comment:
Not relevant because no cookie data is used for this entire controller. Even if you hijack a cookie, there is no change to the security conditions. But we should keep this alert in mind just in case the notifications controller is ever updated to make use of the user cookie.

Reply:
Also, we identify users by a cryptographically secure token that they send to us. Not via sessions.